server/controllers/apply/applicationsController.test.ts

```diff
@@ -128,6 +128,87 @@ describe('applicationsController', () => {
       expect(getSearchOptions).toHaveBeenCalledWith(request, ['crnOrName', 'status'])
       expect(getPaginationDetails).toHaveBeenCalledWith(request, paths.applications.dashboard({}), searchOptions)
     })
+    it('if sort by and sort direction not provided, calls the dashboard service with the sort direction desc', async () => {
+      const searchOptions = createMock<ApplicationDashboardSearchOptions>()
+      const paginatedResponse = paginatedResponseFactory.build({
+        data: applicationFactory.buildList(2),
+      }) as PaginatedResponse<ApprovedPremisesApplicationSummary>
+
+      const paginationDetails = {
+        hrefPrefix: paths.applications.dashboard({}),
+        pageNumber: 1,
+      }
+
+      applicationService.dashboard.mockResolvedValue(paginatedResponse)
```

Comment:
it would improve the clarity here if we used `jest-when` like we do here: https://github.com/ministryofjustice/hmpps-approved-premises-ui/blob/1c246008d83b48c32398507b194fd92876486858/server/controllers/tasksController.test.ts#L59 
It helps to show what we're expecting the function to be called with and preserves type safety